src/main/java/core/basesyntax/exceptions/NoSuchElementException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exceptions;
+
+public class NoSuchElementException extends java.util.NoSuchElementException {
```

Comment:
delete this class and use java.util.NoSuchElementException

Reply:
OK